src/sentry/api/endpoints/organization_events_measurements_histogram.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import absolute_import
+
+import sentry_sdk
+
+from sentry import features
+from rest_framework import serializers
+from rest_framework.response import Response
+
+from sentry.api.bases import OrganizationEventsV2EndpointBase, NoProjects
+from sentry.snuba import discover
+
+# The maximum number of measurements allowed to be queried at at time
+MAX_MEASUREMENTS = 4
+
+
+class MeasurementsHistogramSerializer(serializers.Serializer):
+    num_buckets = serializers.IntegerField(min_value=1)
+    precision = serializers.IntegerField(default=0, min_value=0, max_value=4)
+    measurement = serializers.ListField(allow_empty=False, max_length=4)
```

Comment:
```suggestion
    measurement = serializers.ListField(allow_empty=False, max_length=MAX_MEASUREMENTS)
```

Reply:
Ah my bad, missed it when I was refactoring 👍